test/test_grocy.py

```diff
@@ -345,30 +345,29 @@ def test_get_last_db_changed_invalid_no_data(self):
     @responses.activate
     def test_add_product_valid(self):
         responses.add(responses.POST, f"{CONST_BASE_URL}:{CONST_PORT}/api/stock/products/1/add", status=200)
-        self.assertIsNone(self.grocy.add_product(1, 1.3, 2.44, datetime.now()))
+        self.assertIsNone(self.grocy.add_product(1, 1.3, 2.44, datetime.strptime("2019-05-04 11:31:04", '%Y-%m-%d %H:%M:%S')))
```

Comment:
I guess you could extract this into a global variable instead of creating a new instance all the time.

Reply:
Sure. And the base url format too.